testing/sdk_marathon.py

```diff
@@ -88,6 +88,22 @@ def wait_for_response():
     return config
 
 
+def is_app_running(app: dict) -> bool:
+    return app['tasksStaged'] == 0 and app['tasksUnhealthy'] == 0 and app['tasksRunning'] > 0
```

Comment:
If these keys don't exist in the `app` dict, we're going to get some verbose stack traces, but that's probably OK.

Reply:
https://github.com/mesosphere/dcos-commons/pull/2528/commits/53475bc91153c6abbdb00fc38277b9b8d99654ba